app/javascript/sdk/sdk.js

```diff
@@ -23,6 +23,7 @@ export const SDK_CSS = `
 
 .woot-widget-holder iframe {
   border: 0;
+  color-scheme: normal;
```

Comment:
Added the `color-scheme: normal;` in styles instead of adding it inline because here we can easily override. So if any one can override it by adding `!important`. 
cc @scmmishra